pd/src/main.rs

```diff
@@ -61,6 +61,9 @@ enum Command {
         /// Number of blocks per epoch.
         #[structopt(short, long, default_value = "60")]
         epoch_duration: u64,
+        /// Number of epochs before unbonding stake is released.
+        #[structopt(short, long, default_value = "60")]
+        unbonding_epochs: u64,
```

Comment:
When generating a testnet, we now need to specify how many epochs unbonding takes, or the default will be used. @zbuc is this a good default?

Reply:
60 will coincide with the epoch duration. I think that makes sense as a default for now, but for mainnet we'll likely want different values for both.

Reply:
coinciding isn't a problem, right? like with these defaults, the unbonding period is 3600 blocks, correct?